frontend/src/Nav.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import Results from "./pages/results";
+import App from "./App";
```

Comment:
I think it makes more sense to put navbar in App rather than App in navbar

Reply:
App.js is being used as our home page. We have nav bar in index.js where we can access it on every page.
App is in nav bar so we can route to get to the homepage/App